datasets/my/investor_alert_list/crawler.py

```diff
@@ -0,0 +1,56 @@
+import json
+from zavod import Context, helpers as h
+
+
+def crawl_item(input_dict: dict, context: Context):
+
+    name = input_dict.pop("name")
+
+    # If it's a potential clone, we remove the "potential clone" from the name
+    name = name.replace("Potential clone entity – ", "")
```

Comment:
low prio, but it'd be cool to add `Potential clone entity` to the description if it was present in the name.